app/config/constants.js

```diff
@@ -6,4 +6,7 @@ export default {
   SERVICE_PROVIDERS: {
     MOCK_LOGIN: 'https://github.com/lblod/mock-login-service',
   },
+  MAILING_LIST: {
```

Comment:
Dit bedoelde in omgekeerd. Er zijn verschillende publicatiekanalen (Belga, Vlaanderen.be, …) waarvan ‘MAILING_LIST’ er een is. Zorg dat de syntax ook in lijn is met de rest van de file.